src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -17,5 +17,6 @@ CA1854 | Performance | Info | PreferDictionaryTryGetValueAnalyzer, [Documentatio
 CA1855 | Performance | Info | UseSpanClearInsteadOfFillAnalyzer, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1855)
 CA2019 | Reliability | Info | UseThreadStaticCorrectly, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2019)
 CA2259 | Usage | Warning | UseThreadStaticCorrectly, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2259)
+CA2260 | Usage | Warning | UseCuriouslyRecurringTemplatePatternCorrectly, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2260)
```

Comment:
`UseCuriouslyRecurringTemplatePatternCorrectly` => `ImplementGenericMathInterfacesCorrectly`

Reply:
Strange, I rerun the generated code, not sure why this is not updated